### ISSUE TITLE
file's mime type defaulting to documents

### ISSUE THREAD

**Issue by femee24** [prasmussen/gdrive#129]
Hi
- I tried to upload a csv file using gdrive but it keeps defaulting to document instead of spreadsheet
- I tried the normal command [gdrive upload] and still the same
- I even use the options --mime (force mime type) still no difference

Can you help me to upload my csv file from linux to automatically convert it to "application/vnd.google-apps.spreadsheet" or google sheet.

Thank You

**Comment by JayBrown** [comment]
Seems to be related: https://github.com/prasmussen/gdrive/issues/118

Try uploading the csv manually in your browser & check the mime type. In my case, using a signature file, the mime type was correctly set to application/octet-stream, which means that gdrive seems to be the problem.

**Comment by femee24** [comment]
Hi thank you. I had tried to resolve it by comparing the scenario between my mac and my linux. if i upload  a file on my linux, the file would only be available on google docs but if i upload it using my mac i can open this file on google apps available.

but the options "--mime" doesn't work for spreadsheet it only forces the file to be its original mime

anyway I made an alternatives to this issue. thank you so much
